### PR TITLE
Feature/switch component

### PR DESCRIPTION
**Pull request by amkajaria**

### CODE REVIEW

File: library/src/core/atoms/Switch/Switch.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.slider:before {
+  position: absolute;
+  content: '';
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: 0.4s;
```
Comment (swapnildubal) [inline library/src/core/atoms/Switch/Switch.css]: no need to add css prefixes, we have a package which does it automatically

File: library/src/core/atoms/Switch/Switch.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  const [isChecked, toggleCheck] = useState(checked);
+  const clickHandler = (e) => {
+    toggleCheck(!isChecked);
+    handleToggle(e, isChecked);
+  };
+
+  return (
+    <div className={clsx(classname, 'switch-component')}>
+      <Label className="switch" htmlFor="input-switch">
+        <input
```
Comment (swapnildubal) [inline library/src/core/atoms/Switch/Switch.js]: add accessibility tags

File: library/src/core/atoms/Switch/Switch.css (hunk 1 above)
Comment (amkajaria) [reply library/src/core/atoms/Switch/Switch.css]: removed css prefixes

File: library/src/core/atoms/Switch/Switch.js (hunk 2 above)
Comment (amkajaria) [reply library/src/core/atoms/Switch/Switch.js]: added aria tags

**Pull request merged**